internal/handler/web/studies.go

```diff
@@ -16,6 +17,24 @@ import (
 	"github.com/ucl-arc-tre/portal/internal/types"
 )
 
+func mustParseTime(dateStr string) time.Time {
```

Comment:
nit: `mustParseDate` is more clear

Reply:
[updating variable name](https://github.com/ucl-arc-tre/portal/pull/312/commits/757d960b0562bad7e14699b43baf26407b86d3d0)